builder/main.py

```
@@ -297,6 +557,419 @@ def check_lib_archive_exists():
+        if file_count == 0:
+            print("\nNo files were extracted.")
+            print("The filesystem may be empty or freshly formatted.")
+        else:
+            print(f"\nSuccessfully extracted {file_count} file(s) to {unpack_dir}")
+        
+        return 0
+    except Exception as e:
+        print(f"\nError during extraction: {e}")
+        try:
+            fs.unmount()
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)